addons/l10n_kz/data/account_chart_template_data.xml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+    <record id="chart_template" model="account.chart.template">
+        <field name="name">План счетов</field>
+        <field name="code_digits">4</field>
+        <field name="bank_account_code_prefix">1030.</field>
+        <field name="cash_account_code_prefix">1010.</field>
+        <field name="transfer_account_code_prefix">1020.</field>
```

Comment:
remove dots from the prefixes

Reply:
Why? Subaccounts are made via dots in post-soviet countries